neverbleed.c

```diff
@@ -505,10 +527,39 @@ void neverbleed_transaction_write(neverbleed_t *nb, neverbleed_iobuf_t *buf)
     iobuf_transaction_write(buf, thdata);
 }
 
+static void _exdata_free_callback(void *parent, void *ptr, CRYPTO_EX_DATA *ad, int idx, long argl, void *argp)
```

Comment:
```suggestion
static void do_exdata_free_callback(void *parent, void *ptr, CRYPTO_EX_DATA *ad, int idx, long argl, void *argp)
```
This is nothing more than an editorial preference, but I think we use `do_` to mark functions that are not expected to be called directly.